1.1.is-unique.js

```diff
@@ -1,5 +1,28 @@
 /**
  * 1.1 Is Unique
  * 
- * Implement an algorithm to determine if a string has all unique characters. What if you cannot use additional data structures?
- */
\ No newline at end of file
+ * Implement an algorithm to determine if a string has all unique characters. 
+ * What if you cannot use additional data structures?
+ 
+--> my Input is a string
+--> my Ouput is a boolean
+ */
+
+function isUnique(string){
+    // setup a empty obj and check each character and store 
+    let obj = {};
```

Comment:
Use `const` instead. It's not wrong to use `let` but as the actual value of the object will never change on this function it makes more sense to use `const`. 

Remember that when you assign an object to a variable, the value saved on the variable is a reference (address) to the object in memory.

You would use `let` if you were to change the actual object. Example:

```
let obj = {a: 1};
obj = {b:2};
```